src/tools/gxpkg.ss

```diff
@@ -576,6 +575,28 @@ END
 END
 )
 
+(def Makefile-template #<<END
+ARCH := \\$(shell uname -m)
+DOCKER_IMAGE := "gerbil/gerbilxx:\\$(ARCH)"
+default: linux-static
+
+build:
```

Comment:
lets call it build-release, or have a separate rule for it so that you only need exe: buil spec